src/EventStore.Core/Services/Monitoring/Stats/QueueStats.cs

```diff
@@ -5,18 +5,57 @@ namespace EventStore.Core.Services.Monitoring.Stats
 {
     public class QueueStats
     {
+        ///<summary>
+        ///Queue name
+        ///</summary>
         public readonly string Name;
+        ///<summary>
+        ///Group queue is a member of
+        ///</summary>
         public readonly string GroupName;
+        ///<summary>
+        ///Number of items in the queue
+        ///</summary>
         public readonly int Length;
+        ///<summary>
+        ///The highest number of items in the queues
+        ///</summary>
         public readonly long LengthLifetimePeak;
+        ///<summary>
+        ///The highest number of items in the current queue
+        ///</summary>
         public readonly long LengthCurrentTryPeak;
+        ///<summary>
+        ///The number of milliseconds elapsed processing the current item
+        ///</summary>
         public readonly TimeSpan? CurrentItemProcessingTime;
+        ///<summary>
+        ///Number of milliseconds elapsed since queue went idle
```

Comment:
```suggestion
        ///Time elapsed since queue went idle
```

Reply:
@shaan1337 If the time is in ms then keep my change, if not, then let's specify what it is.

Reply:
@ChrisChinchilla the `TimeSpan` class is a generic class to store time intervals, so you can for example call mytimespan.Seconds or mytimespan.Milliseconds to get it in different units